build.sbt

```diff
@@ -41,7 +41,9 @@ inThisBuild(Def.settings(
     "-language:postfixOps",
   ),
   javacOptions ++= Seq(
-    "-encoding", "UTF-8"
+    "-encoding", "UTF-8",
+    "-source", "1.8",
```

Comment:
Should we add that for Scala as well?

Reply:
would be good to add `-target:jvm-1.8`, though at least on scala12 this is the default (not completely sure about scala13 but it seems @dwijnand tested?)

Reply:
I didn't test it, but it defaults to jvm-1.8 for Scala 2.13 too.

Btw, in Scala 2.11 it defaults to jvm-1.6, so the Scala classfiles would target Java 6 and the Java classfiles would target Java 8.